lib/webmock/request_signature.rb

```diff
@@ -35,11 +35,11 @@ def eql?(other)
     alias == eql?
 
     def url_encoded?
-      !!(headers && headers['Content-Type'] == 'application/x-www-form-urlencoded')
+      !!(headers && headers['Content-Type'].start_with?('application/x-www-form-urlencoded'))
```

Comment:
What happens if there is no Content-Type header? Maybe it would be better to make it nil-safe like this: 
```suggestion
      !!(headers && headers['Content-Type']&.start_with?('application/x-www-form-urlencoded'))
```

Reply:
You're absolutely right, I missed this case 👍 

I hope the usage of safe navigation is okay (it isn't used anywhere in the project yet) - if not, I can change it to use more `&&` chaining 😉